spec/registry.md

```diff
@@ -202,6 +204,12 @@ are encouraged to track development of these options during Tech Preview:
    - `short` (default)
    - `narrow`
 
+The support for the `offset` option is optional.
+The _resolved value_ of the _expression_ is determined by first subtracting the numeric value of `offset` from the  _operand_,
+and then resolving the _expression_ on the calculated value.
```

Comment:
This is the trouble with using :integer (or :number) for both formatting and selection.

So selection you need both the offset, and the original value.
Because you test the exact values against the original value, and the keywords against the value - offset.

But for formatting the value - offset is enough, and you can drop the offset from the resolved options.

Which means that if one does
```
.input {$count :integer offset=2}
.match $count
...
```
then the resolved value for `$count` should have the `offset`.

If we separated the `:input` / `:number` from `:plural` this would have an easy answer: `:plural` keeps the resolved value and resolved options "as is", and `:integer` resolves the value to the value - offset and remove the offset from the resolved values.

I am not pushing to revert that decision.
We voted and all.

But I am open to suggestions, because I am not sure what to do here.

Reply:
Idea: leave the resolved value as is, and leave the offset in the resolved options.
Then the selection can see both.
And the "formatted value" (which we don't have a concept of, although it would be useful) can do the diff.

Trouble is that without the concept of "formatted value" we can't fully describe how the formatting works.

We have a section named "Numeric Value Selection and Formatting", but we don't seem to describe the formatting part at all.

Reply:
My view is that the :offset affects selection and formatting (in parallel), but it is up to us to define _in the :number specification_ for `offset` **how** successive :number offset values are combined  — which does not prevent any other function options from combining in a different way.

Take the following:

```
.local $a {10 :number offset=2}
.local $b {$a :number offset=1}
.match $b
// please forgive any syntax errors
```
It is up to us to determine whether :number offset behaves like:

`.local $a {10 :number :offset=3} // accumulates
`
or

`.local $a {10 :number :offset=1} // replaces
`

I think we should have a general rule that unless otherwise stated, combinations of options do replacement, but that the specification for any given function option can override that behavior.